test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -223,8 +225,6 @@ AndroidArgs
             assert(projectId, "mockProjectId")
 
             // AndroidGcloudYml
-            assert(appApk, appApk)
-            assert(testApk, testApk)
```

Comment:
I reverted these deletions.

Reply:
oh, I forgot to comment about this diff.
That assert doesn't make sense to me, what are you testing?

Reply:
Thanks for commenting, I fixed that here: https://github.com/TestArmada/flank/pull/389/commits/29670fe1864d306af2df4c9b06a3b46b37c1f989

The intent is to test the expected values are present on `androidArgs`. The name was shadowed. I capture the values above the `with` statement to resolve the conflict.